src/dev/mod.ts

```diff
@@ -83,6 +103,7 @@ export async function generate(directory: string, manifest: Manifest) {
 // This file SHOULD be checked into source version control.
 // This file is automatically updated during development when running \`dev.ts\`.
 
+import config from "./deno.json" assert { type: "json" };
```

Comment:
A `./deno.json` implies that fresh expects to be in the root directory of the repository. This might not always be the case, for example in a mono repo that contains multiple clients and/or servers.

It would be great to not depend on a `deno.json` at all, or at least make its path configurable.